src/waveWorker.js

```diff
@@ -14,9 +14,14 @@ global['onmessage'] = function( e ){
     case 'done':
       if (recorder) {
         recorder.requestData();
+        recorder = null;
```

Comment:
What is this for?

Reply:
Allows the GC to recover the memory for the wav file (recordedBuffers, etc.), in case you don't actually destroy the worker for a while.